manifest.base.yml

```diff
@@ -1,13 +1,14 @@
 ---
 buildpack: python_buildpack
-command: python manage.py migrate --fake-initial && waitress-serve --url-prefix /regulations --port=$VCAP_APP_PORT fec_eregs.wsgi:application
+command: python manage.py migrate --fake-initial && gunicorn fec_eregs.wsgi:application
```

Comment:
Just the defaults so far, but if you find some more optimal configs, we'd carry them over to the other projects.